Appraisals

```diff
@@ -17,6 +17,11 @@ appraise "rack_1" do
   gem "rack-test", ">= 0.6"
 end
 
+appraise 'rails_7-0' do
+  gem 'railties', '~> 7.0.0.a'
+  gem 'dalli', '~> 3.0'
```

Comment:
Why is that we're adding `dalli` explicitly on this `rails 7` appraise?

Not consistent to how we are testing the rest of the rails versions.

Reply:
Because otherwise dalli isn't included and the memcache+dalli tests are skipped on Rails 7, but the specific behavior change we're testing and correcting for is in Rails 7's use of memcache+dalli.

Reply:
Gotcha.

The thing is we're not using `rails` appraisals for testing integration with the rest of the gems.

That's what `dalli3` appraisal will eventually be for, when rails 7 is out of the beta. If you check `gemfiles/dalli3.gemfile.lock` right now is testing with `activesupport` 6.1, but will switch to `7.0` when final release.

In the meantime I prefer we add an additional separate appraisal just for the purpose of testing against rails beta and call it out with a comment that could be removed once rails 7 is final.

Reply:
Updated myself.